spine-c/include/spine/IkConstraint.h

```diff
@@ -51,14 +51,17 @@ typedef struct spIkConstraint {
 	int bendDirection;
 	float mix;
 
+	int level;
```

Comment:
Good point, but it looks like it will make code too complicated with the small advantage. But if you think it should be done then I will try.